swagger.json

```diff
@@ -1876,6 +1876,53 @@
                 }
             }
         },
+        "/fleet/drivers/{driver_id}/documents": {
+            "parameters": [
+                {
+                    "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                    "name": "driver_id",
+                    "in": "path",
+                    "required": true,
+                    "description": "ID of the driver to create a document for.",
```

Comment:
super-nit: Personally not a big a fan of ending the sentence with a preposition in the documentation, but I'm fine with leaving it as it is for now.